src/generic.jl

```diff
@@ -0,0 +1,56 @@
+function LinearAlgebra.lu(A::CuSparseMatrixCSR{T,Cint}) where T <: BlasFloat
+  n = LinearAlgebra.checksquare(A)
+  solver = CudssSolver(A, 'G', 'F')
+  x = CudssMatrix(T, n)
+  b = CudssMatrix(T, n)
+  cudss("analysis", solver, x, b)
+  cudss("factorization", solver, x, b)
+  return solver
+end
+
+function LinearAlgebra.ldlt(A::CuSparseMatrixCSR{T,Cint}) where T <: BlasFloat
```

Comment:
I am not sure if we should use `ldlt` here, if cudss is using the Bunch-Kaufman factorization for symmetric indefinite matrix. I would suggest renaming this function to `LinearAlgebra.bunchkaufman`.

Reply:
NVIDIA answered me this evening and they say that it's an LDL' factorization but I don't understand why we can solve a linear system with [0 1; 1 0] in that case. I will have more news tomorrow.

Reply:
ok, good to know